zipline/algorithm.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
         self.algoscript = kwargs.pop('script', None)
 
         self._initialize = None
+        self._analyze = kwargs.pop('analyze', None)
```

Comment:
I should have checked this more closely.  Perhaps a test is warranted.  

I had it working from the elif previously, but moved per a previous suggestion.  Perhaps I misinterpreted the exact location.  

I will update and resubmit a PR later tonight.  

By the way is there a specific branch I should use for the PR?

Thanks.